packages/components/src/Card/Card.tsx

```diff
@@ -86,5 +64,40 @@ export function Card({ accent, children, url, onClick }: CardPropOptions) {
     props.tabIndex = 0;
   }
 
-  return <Tag {...props} />;
+  return (
+    <Tag {...props}>
+      {title && <Title title={title} />}
+      {children}
+    </Tag>
+  );
+}
+
+interface TitleProps {
+  /**
+   * The title for the card.
+   */
+  readonly title: string;
+}
+
+/**
+ * **DEPRECATED** Use `title` props on the `<Card />` to add a title instead.
+ *
+ * This will be used as an internal component
+ */
+
+export function Title({ title }: TitleProps) {
```

Comment:
Since this is already a breaking change do you think we should stop exporting this?

Reply:
That would really break things.. but I guess that's fine because webpacker would tell us about it